lib/cluster.js

```diff
@@ -261,7 +261,7 @@ function masterInit() {
     assert(schedulingPolicy === SCHED_NONE || schedulingPolicy === SCHED_RR,
            'Bad cluster.schedulingPolicy: ' + schedulingPolicy);
 
-    var hasDebugArg = process.execArgv.some(function(argv) {
+    var hasDebugArg = process.execArgv.some(function hasDebugTest(argv) {
```

Comment:
Thanks for the link. rtrott said

> Anonymous functions that are used as listeners should be named.

Above function is not a listener, it should be es6.

Sorry, I filtered through 60 issue threads this morning, it was one on the test/ folder though.